src/contexts/Network/index.tsx

```diff
@@ -66,6 +68,12 @@ export const NetworkMetricsProvider = ({
                   earliestStoredSession.toString()
                 ),
                 fastUnstakeErasToCheckPerBlock: erasToCheckPerBlock.toNumber(),
+                minimumActiveStake: new BigNumber(
+                  planckToUnit(
+                    new BigNumber(minimumActiveStake),
```

Comment:
This doesn't look right. Did you test this value?

minimumActiveStake.toString() should be put into a BigNumber. It shouldn't be converted to unit here, instead should be in planks.